webapp/components/interop.js

```diff
@@ -716,7 +659,40 @@ class Interop2022 extends PolymerElement {
   }
 
   getBrowserScoreTotal(browserIndex) {
-    return this.getBrowserScoreForFeature(browserIndex, SUMMARY_FEATURE_NAME);
+    return this.totals[browserIndex];
+  }
+
+  getAllYears() {
+    const firstYear = 2021;
+    const currentYear = new Date().getFullYear();
+    const years = [];
+    for (let i = firstYear; i <= currentYear; i++) {
+      years.push(i.toString());
+    }
+    return years;
+  }
+
+  updateYearInfo(year) {
+    const yearInfo = PARAMS_BY_YEAR[year];
+    this.focusAreas = yearInfo.focus_areas;
+    this.summaryFeatureName = yearInfo.summary_feature_name;
+    this.tableSections = yearInfo.table_sections;
```

Comment:
`tableSections`, `prose`, and `issueURL` are all used in the template (`tableSections` is used [here](https://github.com/web-platform-tests/wpt.fyi/blob/8e01960ba2a93ecff4b6167161aa1a43f20c4ff6/webapp/components/interop.js#L410) for instance)

Reply:
I see! In that case, these variables should be declared in the `properties()`

Reply:
Also since year is directly passed in from the parent DOM, we don't need the observer method for `updateYearInfo`. We can move everything here to constructor